### PR TITLE
Added Module.Core.Tests

### PR DESCRIPTION
**Pull request by MaherJendoubi**

### CODE REVIEW

**Comment by MaherJendoubi** [comment]
@thiennn Please review. Thanks!

File: test/SimplCommerce.Module.Core.Tests/Components/DefaultAddressViewComponentTest.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            var mockWorkContext = new Mock<IWorkContext>();
+            var user = new User { DefaultBillingAddressId = 0, Id = 1, FullName = "Maher" };
+            mockWorkContext.Setup(x => x.GetCurrentUser()).Returns(Task.FromResult(user));
+            var component = new DefaultAddressViewComponent(mockRepository.Object, mockWorkContext.Object);
+
+            // Act
+            var result = await component.InvokeAsync();
+
+            // Assert
+            Assert.NotNull(result);
```
Comment (thiennn) [inline test/SimplCommerce.Module.Core.Tests/Components/DefaultAddressViewComponentTest.cs]: I think we should have more Assert here to make the Test more valuable. For example:

var viewResult = Assert.IsType<ViewViewComponentResult>(result);
Assert.Null(viewResult.ViewName);
var model= Assert.IsType<DefaultAddressViewComponentVm>(viewResult.ViewData.Model);
Assert.Equal(address.AddressLine1 ,model.Address.AddressLine1);

**Comment by MaherJendoubi** [comment]
@thiennn I wrote a failed test but all checks have passed!

**Comment by MaherJendoubi** [comment]
@thiennn Please review. Thanks!

**Pull request merged**